### PR TITLE
WEBUMENIA-1418 Do not trigger an Elasticsearch error on high page numbers

### PR DESCRIPTION
**Pull request by eronisko**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (link to Jira or GitHub issue)
Closes #362 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:

- [x] I have updated the [CHANGELOG](../CHANGELOG.md)
- [x] I have requested at least 1 reviewer for this PR
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: tests/RefreshSearchIndex.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Tests;
+
+use App\Elasticsearch\Repositories\AuthorityRepository;
+use App\Elasticsearch\Repositories\ItemRepository;
+use Elasticsearch\Common\Exceptions\Missing404Exception;
+
+trait RefreshSearchIndex
```
Comment (rastislav-chynoransky) [inline tests/RefreshSearchIndex.php:9]: In context of Elasticsearch I'd expect it only [refreshes](https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-refresh.html) the index, but I guess you've followed naming of RefreshDatabase here.

**Review by rastislav-chynoransky — APPROVED** [review]

File: tests/RefreshSearchIndex.php (hunk 1 above)
Comment (eronisko) [reply tests/RefreshSearchIndex.php:9]: >  I guess you've followed naming of RefreshDatabase here.

Yeah, exactly. Happy to use a different name, though. Something like `RecreateSearchIndex`?

**Review by igor-kamil — APPROVED** [review]
nice

**Comment by eronisko** [comment]
I'm waiting to merge this in until we get the tests up and runnig again (e.g. #413)

**Pull request merged**